src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/PropertyPages/DebugPageControl.xaml

```diff
@@ -541,12 +551,33 @@
                 IsChecked="{Binding SqlDebugging, Mode=TwoWay}"
                 Margin="9,7,3,7"
                 VerticalAlignment="Center"
+                VerticalContentAlignment="Center"     
+                Content="{x:Static Member=local:PropertyPageResources.chkSqlCodeDebuggingText}">
+                <!-- Disable CheckBox if WebView2 debugging is selected, since we can only launch JS debugger alone -->
+                <CheckBox.IsEnabled>
+                    <MultiBinding Converter="{StaticResource MultiValueBoolToBoolAnd}">
+                        <Binding Path="IsProfileSelected" Mode="OneWay"/>
+                        <Binding Path="JSWebView2Debugging" Mode="OneWay" Converter="{StaticResource InverseBooleanConverter}"/>
+                    </MultiBinding>
+                </CheckBox.IsEnabled>
+            </CheckBox>
+
+            <!-- JavaScript WebView2 Debugging -->
+            <CheckBox
+                Grid.Row="11"
+                Grid.Column="0"
```

Comment:
Could you update this to have the checkbox span multiple columns so that it doesn't push the other content so far over? Please also update the screenshot in the PR.

Reply:
We should apply the same change to the other checkboxes, as they may have longer strings in other languages.